tasks/sync-granule/index.js

```diff
@@ -78,6 +81,23 @@ async function download({
   }
 }
 
+/**
+* Set or disable ACL using value supplied from configuration.
+* Defaults to private. ACL value of 'disabled' disables it.
+*
+* @param {string} acl
+* @returns {string | undefined }
+*/
+function setOrDisableAcl(acl) {
+  if (acl === 'disabled') {
+    return undefined;
+  }
+  if (acl) {
+    return acl;
```

Comment:
Do we want to allow a custom ACL? Is there a use case for that? It's easy enough to do but is there any issue with security/compliance? It feels like it could add some confusion/complexity to support. If there's a want/need for it I could totally be persuaded. I'm just haven't seen it mentioned that users are wanting to add their own ACLs.

Reply:
It also looks like (according to Hector's comment) users can't do a `public` ACL. Would it be confusing if a user tried to set a `public` ACL and it just didn't work or tossed an error?

Reply:
👍  looks good in https://github.com/nasa/cumulus/pull/2946/commits/d01be12a10703b75204c5f22ea23328cbacb13aa#diff-2f2b6b18a2b5bb11ee7c3b817ca91da90127269bc36fc33ab72291f8b6ba9c2dL95-L97